src/lib/components/Actions/Button.svelte

```diff
@@ -3,57 +3,71 @@
 	import type { Snippet } from 'svelte';
 	import type { HTMLButtonAttributes } from 'svelte/elements';
 
-	interface Props {
+	interface Props extends HTMLButtonAttributes {
+		active?: boolean;
+		animation?: boolean;
 		block?: boolean;
 		children: Snippet;
 		color?: DaisyColor;
+		disabled?: boolean;
 		full?: boolean;
 		glass?: boolean;
-		outline?: boolean;
-		onClick?: () => void;
-		responsive?: boolean;
+		onclick?: () => void;
```

Comment:
Any reason or standard you had to change the onClick to onclick?

Reply:
Yeah, onclick is the default prop name on elements in svelte, its not the same as with React where it follows JS camel case styling.

Reply:
kk thanks, good to know

Reply:
actually, I think I'm gonig to redo the typing here, there is redundancy